packages/OverflowMenu/stories/OverflowMenu.examples.stories.js

```diff
@@ -0,0 +1,90 @@
+import React from "react";
+import { getStoryName } from "storybook/storyTree";
+import ExampleStory from "storybook/components/ExampleStory";
+import L10n from "@paprika/l10n";
+import { exampleStoryParameters } from "storybook/assets/storyParameters";
+import OverflowMenuExample from "./examples/OverflowMenuExample";
+import OverflowMenuMultiConfirmationExample from "./examples/OverflowMenuMultiConfirmationExample";
+import OverflowMenuDividersExample from "./examples/OverflowMenuDividersExample";
+import OverflowMenuLongestExample from "./examples/OverflowMenuLongestExample";
+import OverflowMenuTriggerExample from "./examples/OverflowMenuTriggerExample";
+import OverflowMenuOnCloseExample from "./examples/OverflowMenuOnCloseExample";
+import OverflowMenuWithCustomClassExample from "./examples/OverflowMenuWithCustomClassExample";
+import OverflowMenu from "../src";
+
+const storyName = getStoryName("OverflowMenu");
+
+export default {
+  title: `${storyName}/Examples`,
+  component: OverflowMenu,
+};
+
+export const basic = () => (
+  <ExampleStory component="OverflowMenu" storyName="Basic" fileName="examples/OverflowMenuExample.js">
+    <OverflowMenuExample />
+  </ExampleStory>
+);
+
+basic.story = exampleStoryParameters;
+
+export const locale = () => (
+  <ExampleStory component="OverflowMenu" storyName="Locale" fileName="examples/OverflowMenuExample.js">
+    <L10n locale="zh">
+      <OverflowMenuExample />
+    </L10n>
+  </ExampleStory>
+);
+
+locale.story = exampleStoryParameters;
+
+export const modals = () => (
+  <ExampleStory component="OverflowMenu" storyName="Modals" fileName="examples/OverflowMenuMultiConfirmationExample.js">
+    <OverflowMenuMultiConfirmationExample />
+  </ExampleStory>
+);
+
+modals.story = exampleStoryParameters;
```

Comment:
While in general, I like the result of the story names in the left side navigation from this PR – the shorter names are easier to scan.

Old:
<img width="263" alt="Screen Shot 2022-01-14 at 4 54 04 PM" src="https://user-images.githubusercontent.com/14944896/149602445-50be18b3-53fe-4cea-a8fb-efb42fdaefc6.png">

New + Improved:
<img width="258" alt="Screen Shot 2022-01-14 at 4 53 39 PM" src="https://user-images.githubusercontent.com/14944896/149602460-eed5567d-5c7e-4f14-9fcc-e45a31746f1d.png">

The new version is derived from the variable names of your stories... so `fullWidthContent` becomes `Full Width Content` in Storybook.  But in this case "Modals" isn't quite cutting it, and the old version "with multiple confirmation modals" is a bit more helpful (though maybe it could still be shorter).  The trick I'd suggest here, is giving some stories a `name` override if you think it's needed to make it clear what the story is about.  

So, something like this could be used:

```js
modals.story = {
  name: "Multiple Confirmations",
  parameters: exampleStoryParameters,
}
```

I'll let you decide where else this idea could be used in this PR.